cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -257,6 +257,17 @@ void Run(const std::vector<string>& bag_filenames) {
     }
 
     bag.close();
+
+    if (FLAGS_keep_running) {
```

Comment:
If today does not work, I can move the VC to tomorrow same time? Did you get my email?